### ISSUE TITLE
Windows installer weirdness

### ISSUE THREAD

**Issue by gzotti** [Stellarium/stellarium#819]
### Expected Behaviour
Installer allows to reset config.ini and solar system data.
Uninstaller has option to remove all user data including custom landscapes, star data, skycultures, ... (should remove the whole $USERDIR)

### Actual Behaviour
Installer has a checkbox to remove all custom landscapes. Sometimes this checkbox is even pre-activated, but I have not found out why. This is prone to errors and makes no sense IMHO. Landscapes are compatible even if made for version 0.10 ten years ago. 

### Steps to reproduce
Run install/uninstall process

### System
* Stellarium version: at least 0.17 onwards
* Operating system: Windows 7/10

Removing (all, not just landscapes!) user data makes indeed sense if users want to get rid of Stellarium completely, else, all add-on data have been compatible or have auto-updated in the past several years. I don't see a scenario where users would install a new version but purposefully want to delete/reset their carefully prepared own landscape panoramas, bookmarks or ocular/instrument collections.

**Comment by alex-w** [comment]
Georg, please check a fresh beta

**Comment by alex-w** [comment]
After reading documents... no exist simple way to get an additional menu with checkboxes for uninstaller, so, it's task for future

**Comment by gzotti** [comment]
OK. For me it was more important to not delete user data accidentally than to allow deletion on uninstall.

**Comment by alex-w** [comment]
I checked twice the behaviour of Windows installer in current master on install/uninstall tasks and he doesn't delete user data. This is interesting, but in 2017 year I added the code for ask user about deleting user data in uninstaller and... I mistakenly deleted this code in 2018 year when I've refactored the iss-file (removing code for call an installer for MSVC Redist) :(

Now I restored the code for confirmation delete user data and formally the issue is should be closed now.

P.S. Let me final check installer on Windows.
